doc/source/how-to/code/release.yml

```diff
@@ -1,63 +1,36 @@
-name: CI
-on:
-  workflow_run:
-    workflows: ["style", "tests", "docs", "build"]
-    types:
-      - completed
-
-env:
-  MAIN_PYTHON_VERSION: 3.9
-
-concurrency:
-  group: ${{ github.workflow }}-${{ github.ref }}
-  cancel-in-progress: true
-
-jobs:
-
-  release:
-    name: Release
-    if: |
-      github.event_name == 'push' &&
-      contains(github.ref, 'refs/tags') && 
-      github.event.workflow_run.conclusion == 'success'
-    runs-on: ubuntu-latest
-  
-    steps:
-      - name: Checkout repository
-        uses: actions/checkout@v3
-
-      - name: Set up Python
-        uses: actions/setup-python@v4
-        with:
-          python-version: ${{ env.MAIN_PYTHON_VERSION }}
-
-      - name: Install dependencies
-        run: |
-          python -m pip install --upgrade pip build twine
-
-      - name: Build artifacts
-        run: |
-          python -m build && python -m twine check dist/*
-
-      - name: Publish to private PyPI
-        env:
-          TWINE_USERNAME: PAT
-          TWINE_PASSWORD: ${{ secrets.PYANSYS_PYPI_PRIVATE_PAT }}
-          TWINE_REPOSITORY_URL: https://pkgs.dev.azure.com/pyansys/_packaging/pyansys/pypi/upload
-        run: |
-          python -m twine upload dist/* \
-
-      - name: Publish to PyPI
-        env:
-          TWINE_USERNAME: __token__
-          TWINE_PASSWORD: ${{ secrets.PYPI_TOKEN }}
-        run: |
-          python -m twine upload --skip-existing ./**/*.whl
-          python -m twine upload --skip-existing ./**/*.tar.gz
-
-      - name: Publish to GitHub
-        uses: softprops/action-gh-release@v1
-        with:
-          files: |
-            ./**/*.whl
-            ./**/*.tar.gz
+release-pypi-private:
+  name: "Release to private PyPI"
+  runs-on: ubuntu-latest
+  needs: [build-library]
+  steps:
+    - name: "Release to the private PyPI repository"
+      if: github.event_name == 'push' && contains(github.ref, 'refs/tags')
+      uses: pyansys/actions/release-pypi-private@v1
+      with:
+        library-name: ${{ env.LIBRARY_NAME }}
+        twine-username: "__token__"
+        twine-token: ${{ secrets.PYANSYS_PYPI_PRIVATE_PAT }}
+
+release-pypi-public:
+  name: "Release to public PyPI"
```

Comment:
Should we make this parallel to the previous name, perhaps "Release to the public PyPI repository"? What is the difference between line 15 and line 19--because now I see that my suggestion is exactly what appears in line 19.

Reply:
The first name refers to the job name while the second is a step name. Since there is only one step in this job, I think we could even get rid of the step name... Anyway, I agree on using the same name for both.